pkg/netutils/netutils.go

```diff
@@ -98,6 +98,9 @@ func NetLimit() *rate.Rate {
 				continue
 			}
 			speed := compile.FindStringSubmatch(fields[1])
+			if len(speed) == 0 {
```

Comment:
In fact, maybe it's better to use `len(speed)<=1`. WDYT?

Reply:
because in my situation, the value of fields is "[Speed  Unknown!]", so the value of sppeed is [], and len(speed) = 0;
when len(speed) = 1, I think the value of fields will be normal, like "1000Mb/s".

Reply:
We take the value of index 1 that needs to ensure that the length is greater than or equal to 2. 

FYI https://golang.org/pkg/regexp/#Regexp.FindStringSubmatch

Reply:
yeah, I got what you mean and modify it, thx.

Reply:
In addition,I think the type of "dev.Name" can be configurable, maybe you can think about it.

Reply:
Yeah, you are right. I think we should try to redesign this method. 
Thanks for your suggestion.